From 8124f458e93ef8b1723ec447157d879fe8c9ac50 Mon Sep 17 00:00:00 2001
From: hari gowtham <hgowtham@redhat.com>
Date: Wed, 21 Oct 2015 12:12:09 +0530
Subject: [PATCH 43/67] Tier/cli: removing warning message for tiering

The warning message for tiering being under experimental staus is removed.

Change-Id: I7d1d535d380b672c70f03ecc0d24a113600ea43f
BUG: 1274411
Signed-off-by: hari gowtham <hgowtham@redhat.com>
Reviewed-on: http://review.gluster.org/12407
Tested-by: NetBSD Build System <jenkins@build.gluster.org>
Reviewed-by: mohammed rafi  kc <rkavunga@redhat.com>
Tested-by: Gluster Build System <jenkins@build.gluster.com>
Reviewed-by: Dan Lambright <dlambrig@redhat.com>
Tested-by: Dan Lambright <dlambrig@redhat.com>
Signed-off-by: Hari Gowtham <hgowtham@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/59986
---
 cli/src/cli-cmd-volume.c |   10 ----------
 1 files changed, 0 insertions(+), 10 deletions(-)

diff --git a/cli/src/cli-cmd-volume.c b/cli/src/cli-cmd-volume.c
index c23f31b..670a597 100644
--- a/cli/src/cli-cmd-volume.c
+++ b/cli/src/cli-cmd-volume.c
@@ -910,21 +910,11 @@ do_cli_cmd_volume_attach_tier (struct cli_state *state,
         int                     parse_error = 0;
         cli_local_t             *local = NULL;
         int                     type = 0;
-        char                    *question = "Attach tier is recommended only "
-                                            "for testing purposes in this "
-                                            "release. Do you want to continue?";
-        gf_answer_t             answer = GF_ANSWER_NO;
 
         frame = create_frame (THIS, THIS->ctx->pool);
         if (!frame)
                 goto out;
 
-        answer = cli_cmd_get_confirmation (state, question);
-        if (GF_ANSWER_NO == answer) {
-                ret = 0;
-                goto out;
-        }
-
         ret = cli_cmd_volume_add_brick_parse (words, wordcount, &options, &type);
         if (ret) {
                 cli_usage_out (word->pattern);
-- 
1.7.1

